src/main.cc

```diff
@@ -125,6 +143,15 @@ static int install(LiteClient& client, const bpo::variables_map& params, aklite:
     install_mode = params.at("install-mode").as<std::string>();
   }
 
+  LocalUpdateSource local_update_source;
```

Comment:
It seems like there is missing change for the pull command. As result the pull and install doesn't work (the last one is supposed to work but it depends on pull which doesn't work).

Reply:
You are right. I've fixed it now.